src/compiler/sys.ts

```diff
@@ -11,8 +11,8 @@ namespace ts {
      */
     /* @internal */
     export function setStackTraceLimit() {
-        if ((Error as any).stackTraceLimit < 100) { // Also tests that we won't set the property if it doesn't exist.
-            (Error as any).stackTraceLimit = 100;
+        if ((Error as any).stackTraceLimit < Infinity) { // Also tests that we won't set the property if it doesn't exist.

```

Comment:
Do not see the point of the check really. That was to allow for higher values. It is not going to be any higher than infinity.

Reply:
Well as the comment states, the condition also fails if the property doesn't exist (i.e. older engines before this became standardized). Agree though that the check should be refactored if its only remaining use is to verify the property exists.